src/applications/personalization/dashboard/components/AuthApplicationSection.jsx

```diff
@@ -1,69 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
I see that this component doesn't appear to be used but I'd feel a little more comfortable if we could trace down when it was orphaned.

Reply:
I'll see what I can find.  Would you prefer I make this deletion a separate PR so that in the future it might make searching for it a bit easier?

Reply:
It was originally created in the `dashboard-beta` app, then it was renamed and moved into `personalization`, but this was reverted.  I also went as far back as commit `92811be079cac99884abc28c2c96da61f08fe2a5` and found that even though it still wasn't used, the same component existed in the `user-profile` app.

So, it looks like this exists as a result of renaming and regrouping apps, but was leftover when the apps were deleted or transitioned into something new.